apps/journeys-admin/src/components/Team/CopyToTeamMenuItem/CopyToTeamMenuItem.spec.tsx

```
@@ -119,6 +118,75 @@ describe('DuplicateJourneys', () => {
+            }
+          ],
+          getJourneyProfile: {
+            __typename: 'JourneyProfile',
+            id: 'journeyProfileId',
+            lastActiveTeamId: 'teamId'
+          }
+        }
+      }))
+    }
+
+  const translateMock = {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'translateMock'. Consider to delete it or rename it.

'translateMock' is defined here:


(lint/suspicious/noRedeclare)